src/components/AddressComponent/AddressComponent.tsx

```diff
@@ -0,0 +1,52 @@
+import { faArrowUpRightFromSquare } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import classNames from 'classnames';
+import { contractAddress } from 'config';
+import {
+  ACCOUNTS_ENDPOINT,
+  getExplorerLink,
+  MvxCopyButton,
+  MvxTrim,
+  useGetNetworkConfig
+} from 'lib';
+
+interface ContractAddressPropsType {
+  address: string;
+  isHeader?: boolean;
+}
+
+export const AddressComponent = ({
+  address,
+  isHeader = false
+}: ContractAddressPropsType) => {
+  const { network } = useGetNetworkConfig();
+  const explorerAddress = network.explorerAddress;
```

Comment:
Can be directly destructured above: `const { network: { explorerAddress } } = useGetNetworkConfig();`